sys/net/gnrc/network_layer/ipv6/static_addr/gnrc_ipv6_static_addr.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2022 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ */
+
+#include "net/gnrc/ipv6/nib.h"
+#include "net/gnrc/ipv6.h"
+#include "net/gnrc/netapi.h"
+#include "net/gnrc/netif.h"
+#include "net/gnrc/rpl.h"
+#include "net/ipv6/addr.h"
+#include "net/netdev.h"
+#include "net/netopt.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+/**
+ * @brief   ID of the upstream interface, set to 0 to attempt auto-select
+ */
+#ifndef CONFIG_GNRC_IPV6_STATIC_ADDR_UPSTREAM
+#define CONFIG_GNRC_IPV6_STATIC_ADDR_UPSTREAM 0
+#endif
+
+/**
+ * @brief   ID of the downstream interface, set to 0 to attempt auto-select
+ */
+#ifndef CONFIG_GNRC_IPV6_STATIC_ADDR_DOWNSTREAM
+#define CONFIG_GNRC_IPV6_STATIC_ADDR_DOWNSTREAM 0
+#endif
```

Comment:
Any reason, why those use the `CONFIG_` prefix but are neither exposed in a header file nor via Kconfig?